src/services/streetView.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+// @ts-ignore
+import asyncLoading from 'react-async-loader';
+
+interface ReactStreetviewProps {
+  apiKey: string;
+  streetViewPanoramaOptions: google.maps.StreetViewPanoramaOptions;
+  onPositionChanged: Function;
+  onPovChanged: Function;
+  position: google.maps.LatLng;
+  googleMaps: any;
+}
+
+class ReactStreetview extends React.Component<ReactStreetviewProps> {
```

Comment:
Why not functional component?

Reply:
please add reference to documentation

Reply:
README  file updated in https://github.com/hasadna/anyway-newsflash-infographics/pull/407